### ISSUE TITLE
Please add the protocol

### ISSUE THREAD

**Issue by Onair-santa** [wiresock/WireSockUI#60]
Please add the protocol AmnesiaWG in to wiresock and wiresockUI, because in Russia WG is already almost blocked, and Wireguard in Amnesia with obsfuction and works great. There are options for installing it on both the computer and the router, but I would like your solution as everything works fine.
Thank you

https://github.com/amnezia-vpn/amneziawg-windows

**Comment by wiresock** [comment]
You can utilize the SOCKS5 handshake obfuscation feature, which functions very effectively in Russia.

**Comment by Onair-santa** [comment]
Have wiresock this function?

**Comment by wiresock** [comment]
Certainly! For detailed information on SOCKS5 Proxy Configuration, you can refer [here](https://www.wiresock.net/wiresock-vpn-client/wiresock-vpn-client-advanced-configuration-parameters/). Additionally, this [post about bypassing Egypt's WireGuard ban](https://www.ntkernel.com/how-to-bypass-egypts-wireguard-ban/) offers valuable insights. The strategy that proved effective in Egypt is also working well in Russia, and the high volume of downloads from Iran indicates its success there too.

**Comment by Onair-santa** [comment]
And still, I ask you to add the implementation of this data in the config. Your solution is more convenient for me than the native Amnesia solutions. Please, if it is not too difficult. It will be useful to many.
Jc = 79
Jmin = 90
Jmax = 1170
S1 = 19
S2 = 29
H1 = 1667693314
H2 = 1470019548
H3 = 878063205
H4 = 1118465777
.......

**Comment by wiresock** [comment]
Current configuration validation supports the AmneziaWG-compatible `J*`, `S*`, `H*`, `Id`, and `Ip` parameters. Closing as implemented.
